ray-operator/config/samples/ray-service.tpu-multi-host.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: ray.io/v1
+kind: RayService
+metadata:
+  name: s-d-tpu
+spec:
+  serveConfigV2: |
+    applications:
+      - name: stable_diffusion
+        import_path: stable_diffusion.stable_diffusion_tpu:deployment
+        runtime_env:
+          working_dir: "https://github.com/ryanaoleary/serve_config_examples/archive/refs/heads/tpu-example.zip"
+          pip:
+            - pydantic<2
+            - google-api-python-client
+            - pillow
+            - diffusers==0.7.2
+            - transformers==4.24.0
+            - flax
+            - ml_dtypes==0.2.0
+            - jax[tpu]==0.4.11
+            - -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
+            - fastapi
+            - pillow
+            - ray[serve,default]==2.9.0
```

Comment:
Why do we need to install Ray Serve here?

Reply:
I don't think it's needed actually, removed in[3b44939](https://github.com/ray-project/kuberay/pull/2198/commits/3b449390808f87876c499aacb31dac657b903191).